azure-pipelines/compare-refdata.yml

```diff
@@ -36,6 +36,16 @@ jobs:
           source activate tardis
           git clone https://github.com/tardis-sn/tardis-refdata.git $(ref.data.home)
         displayName: "Fetch ALL reference data"
+        condition: false
+
+      - bash: |
+          source activate tardis
+          pip install gdown
+          gdown https://drive.google.com/uc\?id\=1LwfZ6MXuV2j9a23uu9HTcoDDFI9gIuf1 -O tardis-refdata.tar.gz
+          tar zxf tardis-refdata.tar.gz
+          mv tardis-refdata $(ref.data.home)
+        displayName: "Fetch reference data (GDrive backup)"
+        condition: true
```

Comment:
Currently I'm thinking about mirroring the reference data repository to avoid dealing with GitHub LFS transfer limits, I'm not sure if we're keeping the GDrive mirror. You're welcome to bring ideas!

Reply:
@morganlee123 @epassaro this was an idea during hackathon and later I did not think much about this. So you suggest doing a pull from the Moria server? I do not have access to that server. I think that would be the right approach like you said.